invenio_app_rdm/theme/assets/scss/invenio_app_rdm/footer.scss

```diff
@@ -8,6 +8,8 @@
 
  #footer {
    color: #fff;
+   position: absolute;
```

Comment:
Hahaha I don't think that's enough ! (I didn't mean for the footer to always be visible at the bottom, but rather to stick it to the bottom if not enough vertical space to make it seem to be sticking to be at the bottom.

I've implemented it in https://github.com/galterlibrary/InvenioRDM-at-NU .

Reply:
So you mean showing at the bottom always? So even if there is content to scroll, the footer will be visible and then u just scroll the content itself no?

Reply:
Hehe, I looked at Zenodo's CSS and thought that was it... 1.5h hour later the ugly truth revealed itself.

I fixed it now though, tried it out on both Zenodo RDM and InvenioRDM and seems to work for all cases/pages.

I think some of the troubles (or at least solving them) came from the fact that some high-level CSS rules coming from Bootstrap and Semantic UI... But there are plans to fix this, so I'm optimistic :)